### DISCUSSION TITLE
Is there a way to know if a trunk can be remotely closed?

### DISCUSSION THREAD

**Discussion by SylvainGa** [timdorr/tesla-api#499] [Q&A]
Trying to add a feature for closing the trunk but only if the trunk can be remotely closed. Is there a way to know that? Is it what can_actuate_trunks means?

Thanks

**Comment by SystemDisc** [comment]
My tesla has `"can_actuate_trunks": true`, but cannot close its trunks automatically. I don't see a way to tell if the trunk can be closed remotely. For now, you might need to just check the model/year.

**Reply by SylvainGa** [reply]
Ok, thanks.
